docker-compose.yml

```diff
@@ -22,10 +22,14 @@ services:
       SHUTTERBUG_URI:
       # Enables logging to stdout instead of file.
       RAILS_STDOUT_LOGGING: "true"
-      #set to https in .env
+      # set to https in .env
```

Comment:
@scytacki I'm guessing I should have left this out of the commit. Is that right?